osu.Game.Rulesets.Taiko/Edit/Blueprints/TaikoSelectionBlueprint.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Graphics;
+using osu.Game.Rulesets.Edit;
+using osu.Game.Rulesets.Objects.Drawables;
+using osuTK;
+
+namespace osu.Game.Rulesets.Taiko.Edit.Blueprints
+{
+    public class TaikoSelectionBlueprint : OverlaySelectionBlueprint
+    {
+        public TaikoSelectionBlueprint(DrawableHitObject hitObject)
+            : base(hitObject)
+        {
+            RelativeSizeAxes = Axes.None;
+
+            AddInternal(new HitPiece
+            {
+                RelativeSizeAxes = Axes.Both,
+                Origin = Anchor.TopLeft
+            });
+        }
+
+        protected override void Update()
+        {
+            base.Update();
+
+            // Move the rectangle to cover the hitobjects
+            var topLeft = new Vector2(float.MaxValue, float.MaxValue);
+            var bottomRight = new Vector2(float.MinValue, float.MinValue);
+
+            topLeft = Vector2.ComponentMin(topLeft, Parent.ToLocalSpace(DrawableObject.ScreenSpaceDrawQuad.TopLeft));
+            bottomRight = Vector2.ComponentMax(bottomRight, Parent.ToLocalSpace(DrawableObject.ScreenSpaceDrawQuad.BottomRight));
```

Comment:
This is a weird way of going about it :P

Reply:
This is plagiarised from `SelectionHandler`. I'd hope it's not a permanent thing but I'm going to be redoing these blueprints either way.